src/data/string/stringFn.ts

```diff
@@ -7,3 +7,17 @@ export const plStringConstructor = (value: PLString): PLString => {
   typeCheck(PLString, value)
   return value
 }
+
+export const replace = (from: PLString, to: PLString, str: PLString): PLString => {
+  return new PLString(str.value.replace(from.value, to.value))
+}
+
+export const replaceRegexp = (regexp: PLString, flags: PLString, to: PLString, str: PLString): PLString => {
```

Comment:
could you please change order of parameters to:
```suggestion
export const replaceRegexp = (flags: PLString, regexp: PLString, to: PLString, str: PLString): PLString => {
```
this way you can define replace function with default flags:
```ts
(let replace-global (replace-regexp "g"))
```